cmd/external-repos/main.go

```diff
@@ -43,15 +47,33 @@ func main() {
 		log.Debug().Msg("Successfully loaded external repositories.")
 	} else if args[1] == "introspect" {
 		if len(args) < 3 {
-			log.Fatal().Msg("Usage:  ./external_repos introspect URL")
+			log.Panic().Err(err).Msg("Usage:  ./external_repos introspect [--force] URL")
+			os.Exit(1)
 		}
-		count, errors := external_repos.IntrospectUrl(args[2])
+		// TODO Quick option implemented, refactor to use some library
+		var url string
+		if len(args) == 4 {
+			if args[2] != "--force" {
+				log.Info().Err(err).Msgf("Expected --force but got '%s'", args[2])
+				log.Panic().Err(err).Msg("Usage:  ./external_repos introspect [--force] URL")
+				os.Exit(1)
+			}
+			forceIntrospect = true
+			url = args[3]
+		} else {
+			url = args[2]
+		}
+		count, errors := external_repos.IntrospectUrl(url, forceIntrospect)
 		for i := 0; i < len(errors); i++ {
 			log.Panic().Err(errors[i]).Msg("Failed to introspect repository")
 		}
 		log.Debug().Msgf("Successfully Inserted %d packages", count)
```

Comment:
```suggestion
		log.Debug().Msgf("Inserted %d packages", count)
```

This message outputs even if introspection is skipped because of the thresholds. Maybe for now we can just remove the "successfully", because that makes it seem like introspection ran.

I'm not sure this card is the place for it, but I feel like the output messaging for introspect could use some refactoring. It can be confusing, especially when there's multiple errors.